r/R/differential_expression.R

```diff
@@ -55,13 +54,19 @@ runDE <- function(req, data){
     message("checking meta.data slot ", str(data@meta.data))
 
     # Compute differential expression
-    result <- FindMarkers(data, group.by = "custom", ident.1 = "base", ident.2 = "background")
+    result <- presto::wilcoxauc(data, assay = "data", seurat_assay = "RNA",group_by="custom")
+    result <- result[result$group=="base",]
+    rownames(result) <- result$feature
+    result <- result[,c("pval","logFC","pct_in","pct_out","padj","auc")]
+    result$logFC <- log2(10^result$logFC)
```

Comment:
Going from base 10 to base 2 because it's easier to interpret the result, and the UI column is log2